lib/index.js

```diff
@@ -840,7 +898,7 @@ MapboxGeocoder.prototype = {
     if (!results.features.length) return;
     var result = results.features[0];
     this._typeahead.selected = result;
-    this._inputEl.value = result.place_name;
+    this._inputEl.value = this.options.version === 'v6' ? result.properties.name : result.place_name;
```

Comment:
We need to concatenate `result.properties.name` and `result.properties.place_formatted` for V6 features

Reply:
We use only name as an input value for search control

<img width="1133" alt="Screen Shot 2023-01-26 at 2 31 58 PM" src="https://user-images.githubusercontent.com/11231287/214825352-21816a11-cc2a-4e2d-8bde-3a0a5a14debf.png">

Reply:
This is specific to the `query` method that allows programmatic triggering of a search (searches for a string and automatically "selects" the first result). The existing implementation uses the full `place_name` string for v5.